docs/6-quick-start.md

```diff
@@ -80,9 +80,13 @@ Make sure you are sure!!
 
 1. Boot the CN. It will boot to the default platform image, which will most
    likely be SmartOS.
-2. Assign a Linux platform, and reboot it
+2. Assign a Linux platform, set kernel args, and reboot:
 
         sdcadm platform assign <platform> <CN UUID>
+
+        sdc-cnapi /boot/<CN UUID> -X PUT -d \
+            '{"kernel_args": {"systemd.unified_cgroup_hierarchy": "0"}}'
+
```

Comment:
I'm not comfortable with this. Requiring this for each CN isn't something that needs to be burdened onto the user.

If Debian uses cgroupv2 by default, why can't we?